client/server/config/parser.js

```diff
@@ -74,5 +65,26 @@ module.exports = function ( configPath, defaultOpts ) {
 	const serverData = Object.assign( {}, data, getDataFromFile( secretsPath ) );
 	const clientData = Object.assign( {}, data );
 
+	// Optionally override API secrets with env values, only for server data
+	serverData.wpcom_calypso_rest_api_key =
+		process.env.WPCOM_CALYPSO_REST_API_KEY ?? serverData.wpcom_calypso_rest_api_key;
+	serverData.wpcom_calypso_support_session_rest_api_key =
+		process.env.WPCOM_CALYPSO_SUPPORT_SESSION_REST_API_KEY ??
+		serverData.wpcom_calypso_support_session_rest_api_key;
+
+	if (
+		data.features &&
+		data.features[ 'wpcom-user-bootstrap' ] &&
+		! (
+			serverData.wpcom_calypso_rest_api_key || serverData.wpcom_calypso_support_session_rest_api_key
```

Comment:
Just one more fixup: for the server site user bootstrap to work, the `wpcom_calypso_rest_api_key` must be defined. The `wpcom_calypso_support_session_rest_api_key` is not a valid alternative. That means we only need to check for `! serverData.wpcom_calypso_rest_api_key` before disabling the flag.

If `wpcom_calypso_support_session_rest_api_key` is missing, then requests with a `x-support-session` header will be failing (throwing inside `getBootstrappedUser`) and that's fine: such an under-configured server simply doesn't support support sessions, which are a very specialized feature anyway.

Reply:
Ah, TIL. Fixed!